### ISSUE TITLE
Suggestion "Sort 

### ISSUE THREAD

**Issue by scholt** [jamerst/AutoTag#21]
Hello there,

again I wanted to suggest a feature, although it's a bit for an edgecase. I'm filling up an iPod with some TV Shows, but somehow the old Software is not able to properly sort the episodes although they are correctly numbered with season and episode number.

To get the right sort order on an iPod (classic) I needed to set the "Titel für Sortierung" or "sort_name" to something similar than S01E01 ...

Dump from "ffprobe -show_entries 'stream_tags : format_tags'":
```
TAG:season_number=1
TAG:episode_sort=15
TAG:episode_id=15
TAG:date=2008
TAG:sort_name=015
```

Cheers
Tobias

**Comment by jamerst** [comment]
That should be possible to add.

It would be helpful if you could provide an example of a file that is tagged correctly (just a few seconds is fine, I only need the tags). The tag names that ffmpeg report are not the same as the ones actually used in the file, so I'm not entirely sure which one `sort_name` refers to.

**Comment by scholt** [comment]
@jamerst hello there, sorry for getting back so late. I've created a sample setup. I shot some seconds long clips, and used the TV app to Tag them as TV Show. 

The problem I referred to, would be that I would see the episodes in order of 2 3 and 1 because the titles are prefixed with a/b/c (a2, b3, c1) so basically the episode title is used as filename and then on very old devices for sorting. I also added the Sort Title Tags like S01E01, S01E02, etc. which usually fix this problem.

Hope these files help :)


https://github.com/jamerst/AutoTag/assets/2310668/d32f3710-af1d-416a-bb91-29742e5f51ba


https://github.com/jamerst/AutoTag/assets/2310668/76bf504e-2f23-45a7-b85d-07d529794e8d


https://github.com/jamerst/AutoTag/assets/2310668/8084b9e9-8ae5-4faa-95f5-042132367b0c
